package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "pretest": "eslint index.js lib test/*.js",
-    "test": "nyc tape test/*.test.js",
-    "coverage": "nyc --reporter html tape test/*.test.js && opener ./coverage/index.html",
+    "test": "nyc mocha test/*.test.js",
```

Comment:
moving from tape to mocha?

I find it almost impossible to work with tape when tests are failing which is 90% of a refactor. The output is really hard to read and I get rather confused about what is happening. Mocha also exits the same way so no matter what caused the test to fail (a bad assert or invalid syntax due to an unexpected result) you see the same report.